champions.html.slim

```diff
@@ -0,0 +1,77 @@
+---
+layout: microsite
+hero_class: champions
+---
+
+= define 'hero'
+  .row
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/roadtoawesome_graphic.png"))
+    .large-12.column
+      img(src=cdn(site.base_url + "/images/champions/jbosschampion_logo.png"))
+      p JBoss Champions is a selected group of community members who are passionate advocate of JBoss technologies under a program sponsored by Red Hat. JBoss Champions use a variety of JBoss technologies and actively share their deep technical expertise about it in the community, within their company and their customers or partners. This could be done in forums, blogs, screencasts, tweets, conferences, social media, whitepapers, articles, books, and other means.
```

Comment:
This overflowing the banner area on my Chrome - see http://screencast.com/t/YErCc6324